.github/workflows/add-lang.yml

```diff
@@ -0,0 +1,73 @@
+name: Add new language
+on:
+  issues:
+    types: [ labeled ]
+jobs:
+  add_new_language:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'new language'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: next-release
+
+# Retrieve po4a installation cache or create it if not found:
+      - name: Cache po4a dependencies
+        uses: actions/cache@v1.0.3
+        id: cache-po4a
+        with:
+          path: "~/po4a"
+          key: ${{ runner.os }}-po4a
+      - name: Install or retrieve po4a from cache
+        env:
+          CACHE_HIT: ${{ steps.cache-po4a.outputs.cache-hit }}
+
+# If CACHE_HIT: true, retrieve the cache. If not, install po4a and its dependencies and copy them all to a folder (~/po4a/) to be cached:
+        run: |
```

Comment:
Since we’re using this three times now, would it be possible to have it in a shell script?

Reply:
Yes, that makes sense. Done.